buildkite/provider.go

```diff
@@ -135,11 +136,10 @@ func New(version string) provider.Provider {
 func Provider(version string) *schema.Provider {
 	provider := &schema.Provider{
 		ResourcesMap: map[string]*schema.Resource{
-			"buildkite_pipeline":              resourcePipeline(),
-			"buildkite_pipeline_schedule":     resourcePipelineSchedule(),
-			"buildkite_team":                  resourceTeam(),
-			"buildkite_team_member":           resourceTeamMember(),
-			"buildkite_organization_settings": resourceOrganizationSettings(),
```

Comment:
This is still draft but one first comment: remember we will need to maintain this name `buildkite_organization_settings` as well until v1 or at least for a little while